ta/pkcs11/src/pkcs11_token.h

```diff
@@ -128,14 +128,21 @@ enum pkcs11_proc_state {
 	PKCS11_SESSION_BUSY,
 };
 
+enum pkcs11_proc_op {
+	PKCS11_OP_INIT = 0,
+	PKCS11_OP_UPDATE,
+	PKCS11_OP_FINAL,
+	PKCS11_OP_FINAL_ONESHOT,
```

Comment:
maybe rename `PKCS11_OP_FINAL_ONESHOT` to `PKCS11_OP_ONESHOT`.

Reply:
Good Idea. I will do that.